src/Microsoft.Azure.SignalR.Common/Microsoft.Azure.SignalR.Common.csproj

```diff
@@ -6,6 +6,7 @@
   
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;netcoreapp3.0;net5.0</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
which code block is unsafe?

Reply:
method `private unsafe static byte[] UnsafeDecode(string str)` in Base64UrlEncoder.cs